### ISSUE TITLE
ungoogled-chromium-119.0.6045.159_p1: does not build with icu-74

### ISSUE THREAD

**Issue by gringus** [PF4Public/gentoo-overlay#280]
There is a problem when trying to build `ungoogled-chromium-119.0.6045.159_p1` with `icu-74` (`icu-74.1` in my case). I used the following patch from portage (I had to adjust the [patch](https://github.com/gentoo/gentoo/blob/master/dev-qt/qtwebengine/files/qtwebengine-6.5.3-icu74.patch) from portage, but I had to adjust path to the file to `chromium/third_party/blink/renderer/platform/text/text_break_iterator.cc`). Keep in mind this is a quick hack to make it build, so a better solution might be needed/available. I tested the patch and it builds just fine - so far I have not noticed any side effects.

**Comment by PF4Public** [comment]
Given that icu-74 is still testing, I'll do nothing and just wait :)

**Comment by PF4Public** [comment]
> It looks like `icu-74` broke many things.. including firefox etc.

Which makes me more eager to wait :D

**Comment by gringus** [comment]
https://bugs.gentoo.org/917645

**Comment by PF4Public** [comment]
@gringus Have you tried https://gitweb.gentoo.org/repo/gentoo.git/commit/www-client/chromium?id=ebf5fd28b663af33956cbda5b68dc758400b51ba?

**Comment by gringus** [comment]
@PF4Public No, but from the look of it, it is the very same patch I sent you earlier and used by `qtwebengine`
